### DISCUSSION TITLE
Multiple GitHub repos?

### DISCUSSION THREAD

**Discussion by mbestes** [thebaselab/codeapp#441] [Q&A] [answered]
Are we able to have multiple GitHub repos at once? I added one, but each time I tap “clone a repo” from the welcome screen it does nothing. If not and we need to remove the “current”, I can do that but not sure how.

**Comment by mbestes** [comment, marked as answer]
Ah, I see. I needed to open up a new folder (/Code root, e.g.), then I can clone a new repo.

**Comment by Hamburda** [comment]
> Ah, I see. I needed to open up a new folder (/Code root, e.g.), then I can clone a new repo.

Can you please tell me how you managedd to use git hub and how you made it work i keep getting the cant get from origin error and idk how to fix it could you please help idk how to link a repo in this app please help me
